### PR TITLE
Add Traditional Chinese (zh-TW) translation

### PR DESCRIPTION
**Pull request by david082321**
BTY, which code can i use in `pupgui2/resources/ctmods/ctmod_*.py` files?

Now i use this: `CT_DESCRIPTION['zh']`

Can i change to `CT_DESCRIPTION['zh-tw']` or `CT_DESCRIPTION['zh_tw']` ? Which is correct?

### CODE REVIEW

File: pupgui2/resources/ctmods/ctmod_00protonge.py [hunk 1]
```diff
@@ -12,6 +12,7 @@
 CT_DESCRIPTION['de'] = '''Steam Kompatibilitätstool für Windows-Spiele mit Verbesserungen gegenüber Valve's Proton.<br/><br/><b>Verwende dies, wenn du dir nicht sicher bist.</b>'''
 CT_DESCRIPTION['nl'] = '''Steam-compatibiliteitshulpmiddel voor het spelen van Windows-games. Dit hulpmiddel bevat diverse verbeteringen ten opzichte van Valve's Proton.<br/><br/><b>Bij twijfel, kies dit hulpmiddel.</b>'''
 CT_DESCRIPTION['pl'] = '''Narzędzie kompatybilności Steam do uruchamiania Windowsowych gier z poprawkami nad Valve'owym domyślnym Protonem.<br/><br/><b>Użyj tego, jeśli nie wiesz co wybrać.</b>'''
+CT_DESCRIPTION['zh'] = '''Steam 用於執行 Windows 遊戲的相容性工具，改善了 Valve 預設的 Proton。<br/><br/><b>當您不知道該選擇什麼時使用此工具。</b>'''
```
Comment (david082321) [inline pupgui2/resources/ctmods/ctmod_00protonge.py:15]: Can i change to `CT_DESCRIPTION['zh-tw']` or `CT_DESCRIPTION['zh_tw']` ? Which is correct?

Comment (DavidoTek) [reply pupgui2/resources/ctmods/ctmod_00protonge.py:15]: First of all, thank you for providing the translation!

The program uses ISO 639 codes and there currently is no distinction, only `zh`. If necessary, I will do some investigation on how to change the behaviour.

Comment (david082321) [reply pupgui2/resources/ctmods/ctmod_00protonge.py:15]: > First of all, thank you for providing the translation!
> 
> The program uses ISO 639 codes and there currently is no distinction, only `zh`. If necessary, I will do some investigation on how to change the behaviour.

Cause China use zh-CN, and had the most people in the world.
Taiwan use zh-TW, this is the language that i want to translate.

for foreseeable future after this translate merge, may have someone want to change translate into zh-CN.

**Comment by DavidoTek** [comment]
I have changed the program. Now it supports long language/country codes.

For example, if the language is `zh-TW`
1. It will first look for a file called `pupgui2_zh_TW.qm` and `CT_DESCRIPTION['zh_TW']`
2. If not found, then it looks for `pupgui2_zh.qm` and `CT_DESCRIPTION['zh']`
3. If none of these is found it will default back to English.

**Comment by david082321** [comment]
> I have changed the program. Now it supports long language/country codes.
> 
> For example, if the language is `zh-TW`
> 
> 1. It will first look for a file called `pupgui2_zh_TW.qm` and `CT_DESCRIPTION['zh_TW']`
> 2. If not found, then it looks for `pupgui2_zh.qm` and `CT_DESCRIPTION['zh']`
> 3. If none of these is found it will default back to English.

Thank for your help. i had update the translate.

**Comment by DavidoTek** [comment]
Thanks!

**Pull request merged**